go/vt/sqlparser/ast.go

```diff
@@ -438,11 +445,13 @@ func (*CreateTable) iStatement()       {}
 func (*CreateView) iStatement()        {}
 func (*LockTables) iStatement()        {}
 func (*UnlockTables) iStatement()      {}
+func (*DropTable) iStatement()         {}
 
 func (*DDL) iDDLStatement()         {}
 func (*CreateIndex) iDDLStatement() {}
 func (*CreateView) iDDLStatement()  {}
 func (*CreateTable) iDDLStatement() {}
+func (*DropTable) iDDLStatement()   {}
```

Comment:
What do `iStatement` and `iDDLStatement` mean?

Reply:
`iStatement` is an empty function that is needed to implement the Statement interface. Without this all the SQLNodes would also implement statement interface which we dont want. `iDDLStatement` is the DDL counterpart of `iStatement`. This just makes it super clear all the statements that implement the DDLStatement interface and no other struct implements them by accident.

Reply:
Check these out https://github.com/vitessio/vitess/blob/777a19dbe6895f8db7de0c7cccfeec279cb0ec66/go/vt/sqlparser/ast.go#L42 and https://github.com/vitessio/vitess/blob/777a19dbe6895f8db7de0c7cccfeec279cb0ec66/go/vt/sqlparser/ast.go#L60

Reply:
Got it, thanks for explaining!